spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/actions/BaseDeleteOrphanFilesSparkAction.java

```diff
@@ -191,9 +196,10 @@ private String jobDesc() {
     List<String> matchingFiles = Lists.newArrayList();
 
     Predicate<FileStatus> predicate = file -> file.getModificationTime() < olderThanTimestamp;
+    PathFilter filter = pathFilter(table.spec());
```

Comment:
It's a bit confusing to me to have both `predicate` and `filter`, especially as variables passed to the same function.

I know `pathFilter` is the name of the function, but would it make sense to name this variable as `pathFilter` and the function as something like `buildPathFilter` or `inferHiddenPathFilter`?